packages/cmcd-validator-library/src/keyValueValidator/validatorFunctions.js

```diff
@@ -61,11 +62,12 @@ export const isPositive = (errors, key, value) => {
   }
 };
 
-export const checkBlKey = (cmcdJson, errors, key, value) => {
-  if (!('ot' in cmcdJson)) {
-    const description = `The '${key}'key should only be sent with the 'ot' key.`;
-    errors.push(createError(errorTypes.invalidValue, key, value, description));
-  }
+export const checkBlKey = (cmcdJson, warnings, key, value) => {
+  if (!('ot' in cmcdJson) || cmcdJson['ot'] !== 'a' && cmcdJson['ot'] !== 'v'
```

Comment:
Instead of writing (cmcdJson['ot'] !== 'a' && cmcdJson['ot'] !== 'v' && cmcdJson['ot'] !== 'av')  it can be simplified with: !['a','v','av'].includes(cmcdJson['ot'])

Reply:
Thanks ! I will change it